File: WebAzureAcitySenario/Models/Tweet.cs
namespace Tweets.DB
{
    public class Tweet  // Changed from 'record' to 'class'
    {
        public int Id { get; set; }  

        public string Name { get; set; } = string.Empty;  // Added default values to avoid null issues

        public string Text { get; set; } = string.Empty;  

        public int Likes { get; set; }  

        public int Dislikes { get; set; }  

        public string Image { get; set; } = string.Empty;  
    }
}

File: Twitter_Sample2/Twitter_Sample2/Tweet.swift
//
//  to.swift
//  Twitter_Sample2
//
//  Created by pk on 2025-01-30.
//


import SwiftUI

struct Tweet: Identifiable, Decodable {
    let id: Int
    let name: String
    let text: String
    let likes: Int
    let dislikes: Int
    let image: String
}
